src/prerender.ts

```diff
@@ -220,7 +220,9 @@ export async function prerender(nitro: Nitro) {
     // Guess route type and populate fileName
     const contentType = res.headers.get("content-type") || "";
     const isImplicitHTML =
-      !route.endsWith(".html") && contentType.includes("html");
+      !route.endsWith(".html") &&
+      !route.endsWith(".json") &&
```

Comment:
We could also add `.xml` perhaps...

Reply:
Yeah, we could make a list also (for like `.webappmanifest`) but consider this is mainly a hotfix for when content-type is also wrongly text/html...

Reply:
A safelist is probably not a bad idea given this is only for _implicit_ content type...